src/modules/dashboard/staff/helpers/privileges.helpers.ts

```diff
@@ -0,0 +1,21 @@
+import { Privileges as PrivilegesEnum } from '@/global/enums/privileges.enum';
+import { TranslateResult } from 'vue-i18n';
+import i18n from '@/plugins/i18n';
+
+// TODO: add translations
+export function getPrivilegeName(
+  privilege: PrivilegesEnum,
+): TranslateResult {
+  switch (privilege) {
+    case PrivilegesEnum.READONLY:
+      return i18n.global.t('Readonly');
+    case PrivilegesEnum.FULL:
+      return i18n.global.t('Full');
+    case PrivilegesEnum.MANAGE:
+      return i18n.global.t('Manage');
+    case PrivilegesEnum.NOT_ALLOWED:
+      return i18n.global.t('Not allowed');
+    default:
+      throw new Error('Unknown privilege');
+  }
```

Comment:
Nie lubię switchy, nie wiem jak Ty :D opcjonalnie

const privilegeNames: Record<PrivilegesEnum, string> = {
    [PrivilegesEnum.READONLY]: 'Readonly',
    [PrivilegesEnum.FULL]: 'Full',
    [PrivilegesEnum.MANAGE]: 'Manage',
    [PrivilegesEnum.NOT_ALLOWED]: 'Not allowed',
  };

  const name = privilegeNames[privilege];
  
  if (!name) {
    throw new Error('Unknown privilege');
  }

  return i18n.global.t(name);